scripts/update_ts.sh

```diff
@@ -194,7 +194,8 @@ EOF
 
 	QT_INSTALL_HEADERS=$(qmake -query QT_INSTALL_HEADERS)
 
-	echo "TR_EXCLUDE = ${QT_INSTALL_HEADERS%
}/*" >>qgis_ts.pro
+	echo "TR_EXCLUDE = ${QT_INSTALL_HEADERS%
```

Comment:
Was this change on purpose? I can't really tell the purpose ...

Reply:
Nope, accident

Reply:
Fixed